sessiontxn/isolation/repeatable_read.go

```diff
@@ -73,7 +74,7 @@ func (p *PessimisticRRTxnContextProvider) getForUpdateTs() (ts uint64, err error
 		return 0, err
 	}
 
-	if p.followingOperatorIsPointGetForUpdate {
+	if p.optimizeForNotFetchingLatestTS {
```

Comment:
I think you should put line 77~80 before line 72("var txn kv.Transaction") to avoid calling " p.activateTxn()" sometimes

Reply:
Actually, if p.activateTxn is called before, it will return directly.